### DISCUSSION TITLE
Precompiled firmware for Hermera + bltouch on e3v2

### DISCUSSION THREAD

**Discussion by synackbar** [Jyers/Marlin#1038] [Ideas]
Any chance you could also add precompiled firmware for those of us with using a e3d hemera + bltouch? I have it running on modified marlin firmware, but yours is so much better. The changes included temp modifications, offsets etc. I used https://e3d-online.zendesk.com/hc/en-us/articles/360017968457-Hemera-Creality-Ender-3-V2-Firmware-Bl-touch- .

**Comment by Jyers** [comment]
If you're compiling your own version already, the that's half your work done! Just use my source code instead of base marlin and start with one of the template configs to get the proper display settings. That should be it :)

**Comment by dkoke** [comment]
I haven't done this yet, because I compiled my own firmware with the help of the same guide you linked. With the Jyers firmware being updated, I will probably have to update soon.

 I'm fairly certain you could use one of the precompiled firmware versions and really the only thing that needs to be changed is the e steps. I'm thinking you can get away with the default thermistor value and min temp value. 

All you should have to do is set the e steps and off set inside the printer menu and save it to the EEPROM.
